### DISCUSSION TITLE
Where is the example for Azure Devops

### DISCUSSION THREAD

**Discussion by JQUINONES82** [Azure/ResourceModules#1474] [General]
https://github.com/azure/ResourceModules/wiki/Solution%20creation

It would be nice to have an example for Azure Devops as there is only a Github reference

**Comment by AlexanderSehr** [comment]
Hey @JQUINONES82, thanks for the request. I'll create an issue for it (#1514). Note however, the example which is there is actually not supposed to be a 'GitHub example' on how to deploy the modules, but 'a' example of a pipeline-orchestration using GitHub. 

I agree that we should have an Azure DevOps counterpart, but 'd recommend to leverage the template-orchestration as much as possible (and deploy it using a pipeline job).

**Comment by eriqua** [comment]
Hi @JQUINONES82, sorry for the late heads-up. An additional section for Azure Devops has been added to the [Solution Creation](https://github.com/azure/ResourceModules/wiki/Solution%20creation#azure-devops-samples) wiki page by @segraef in PR https://github.com/Azure/ResourceModules/pull/1862

**Comment by ranjbassi** [comment]
Hi 
Just have a question on where to define the parameters in the file Register-AzureDevOpsPipeline.ps1

I am using AZDO and have imported the CARML files into my project repo. 

I now want to use this ps1 file to register all the bicep yaml files into pipelines in Azdo. 

I think I have defined the parameters in the file but can't seem to get it working. 

Where exactly do I need to define my org, project etc. Can anyone help me? Thanks
